base/main_test_bucket_pool.go

```diff
@@ -30,67 +30,27 @@ import (
 // GTestBucketPool is a global instance of a TestBucketPool used to manage a pool of buckets for integration testing.
 var GTestBucketPool *TestBucketPool
 
-// Bucket names start with a fixed prefix and end with a sequential bucket number and a creation timestamp for uniqueness
-const (
-	tbpBucketNamePrefix = "sg_int_"
-	tbpBucketNameFormat = "%s%d_%d"
-	tbpScopePrefix      = "sg_test_"
-	tbpCollectionPrefix = "sg_test_"
-)
-
-const (
-	envTestClusterUsername     = "SG_TEST_USERNAME"
-	DefaultTestClusterUsername = DefaultCouchbaseAdministrator
-	envTestClusterPassword     = "SG_TEST_PASSWORD"
-	DefaultTestClusterPassword = DefaultCouchbasePassword
-	envTestClusterDriver       = "SG_TEST_DRIVER"
-
-	// Creates this many buckets in the backing store to be pooled for testing.
-	tbpDefaultBucketPoolSize = 3
-	tbpEnvPoolSize           = "SG_TEST_BUCKET_POOL_SIZE"
-
-	defaultBucketQuotaMB = 200
-	tbpEnvBucketQuotaMB  = "SG_TEST_BUCKET_QUOTA_MB"
-
-	// Prevents reuse and cleanup of buckets used in failed tests for later inspection.
-	// When all pooled buckets are in a preserved state, any remaining tests are skipped instead of blocking waiting for a bucket.
-	tbpEnvPreserve = "SG_TEST_BUCKET_POOL_PRESERVE"
-
-	// Prints detailed debug logs from the test pooling framework.
-	tbpEnvVerbose = "SG_TEST_BUCKET_POOL_DEBUG"
-
-	tbpUseDefaultCollection = "SG_TEST_USE_DEFAULT_COLLECTION"
-
-	// wait this long when requesting a test bucket from the pool before giving up and failing the test.
-	waitForReadyBucketTimeout = time.Minute
-)
-
-// openCollections represent two Collection as Bucket that are the named and unnamed collection time.
-type openCollections struct {
-	namedCollection   Bucket
-	defaultCollection Bucket
+// a dirtyBucket is passed to bucketReadier to be cleaned
+type dirtyBucket struct {
```

Comment:
Can this be removed now that it just contains a single property?  (and bucketReadierQueue can just be chan tbpBucketName)